it/content/src/main/content/jcr_root/content/forms/af/core-components-it/samples/af2-custom-function/basic/.content.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:fd="http://www.adobe.com/aemfd/fd/1.0"
+          xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+          xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+          jcr:primaryType="cq:Page">
+    <jcr:content
+            cq:deviceGroups="[mobile/groups/responsive]"
+            cq:lastModified="{Date}2021-12-02T15:12:59.900+05:30"
+            cq:lastModifiedBy="admin"
+            cq:template="/conf/core-components-examples/settings/wcm/templates/af-blank-v2"
+            jcr:language="en"
+            jcr:primaryType="cq:PageContent"
+            jcr:title="Adaptive Form V2 (IT)"
+            sling:resourceType="forms-components-examples/components/page">
+        <guideContainer
+                fd:version="2.1"
+                fieldType="form"
+                clientLibRef="customFunctionCategory"
+                jcr:primaryType="nt:unstructured"
+                sling:resourceType="forms-components-examples/components/form/container">
+            <textinput
+                    fieldType="text-input"
+                    jcr:primaryType="nt:unstructured"
+                    name="textinput1"
+                    jcr:title="Text Input 1"
+                    sling:resourceType="forms-components-examples/components/form/textinput"/>
+            <text
+                    fieldType="text"
+                    jcr:primaryType="nt:unstructured"
+                    name="text1"
+                    jcr:title="Text 1"
+                    sling:resourceType="forms-components-examples/components/form/text"
+                    value="Custom Text">
+                <fd:rules
+                        jcr:primaryType="nt:unstructured"
+                        visible="!(textinput1 == 'Hide me') || textinput1 == 'Show me'"/>
```

Comment:
Can you write a rule in visual editor with custom function being invoked ? You can do it here